pkg/serviceaccounts/oauthclient/oauthclientregistry.go

```diff
@@ -73,14 +203,117 @@ func (a *saOAuthClientAdapter) GetClient(ctx kapi.Context, name string) (*oautha
 		// TODO update this to allow https redirection to any
 		// 1. service IP (useless in general)
 		// 2. service DNS (useless in general)
-		// 3. route DNS (useful)
-		// 4. loopback? (useful, but maybe a bit weird)
-		RedirectURIs: redirectURIs,
+		// 3. loopback? (useful, but maybe a bit weird)
+		RedirectURIs: sets.NewString(redirectURIs...).List(),
 		GrantMethod:  a.grantMethod,
 	}
 	return saClient, nil
 }
 
+func parseModelsMap(annotations map[string]string) map[string]model {
+	models := map[string]model{}
+	for key, value := range annotations {
+		if prefix, name, ok := parseModelPrefixName(key); ok {
```

Comment:
Wouldn't this be inconsistent with being the short-if :cop: ?  I find the use of a `continue` to be less clear...